### PR TITLE
Leaflet migration

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: static/js/utils.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 function alert(type, message, delay) {
   console.info(type, message)
+  if(!message) {
```
Comment (vinyll) [inline static/js/utils.js]: mhhh… why would you call alert with only 1 argument?
What would be the type?

File: static/js/utils.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+    message = type
+  }
   let notification = document.querySelector('#notification')
```
Comment (vinyll) [inline static/js/utils.js]: `const notification = document.querySelector('#notification') || document.createElement('div')`
But why would `#notification` not be found? Isn't it supposed to be in the layout?
Notification is not just a plain blank `div`, it has attributes.

**Comment by vinyll** [comment]
And what is this commit about changing alert and notifications doing in leaflet migration??

File: static/js/utils.js (hunk 1 above)
Comment (fuentesloic) [reply static/js/utils.js]: because when an user block the location from his browser, the browser run alert as `alert(message)`
so message is `undefined` and type is a message

File: static/js/utils.js (hunk 2 above)
Comment (fuentesloic) [reply static/js/utils.js]: you write this ... :)

**Comment by fuentesloic** [comment]
It's not false :) I put it here because the probleme is appear when the browser send me `alert()` 
error during localization

File: static/js/utils.js (hunk 2 above)
Comment (vinyll) [reply static/js/utils.js]: Well, you let this go through the review without giving an explicit reason? 😛 
It's either not applicable anymore, or it's missing a comment to explain why I needed to do this :/

File: static/js/utils.js (hunk 1 above)
Comment (vinyll) [reply static/js/utils.js]: Are you saying that the browser is calling the `alert()` function by itsself when rejecting geolocation?
It would then make sense, though probably in another PR (as it's not specific to this PR)

**Comment by vinyll** [comment]
> It's not false :) I put it here because the probleme is appear when the browser send me alert()
error during localization

Ok, I see the point.
Notice that the type would still exist and is used for the _classname_. for an `alert('test me now')`, you would end with a `<div class="test me now">test me now</div>`. We probably want to set a class by default (therefore a `type` which may default to "_info_") so the html does not break.

File: static/js/utils.js (hunk 2 above)
Comment (fuentesloic) [reply static/js/utils.js]: I will cut and paste your comments in the new PR

File: static/css/around-map.css [hunk 3]
```diff
@@ -295,63 +295,34 @@ footer {
...
-}
-
-.current-marker {
-  border: 0.4rem solid #3ba77c !important;
-  z-index: 1;
-}
-
 .visible {
-  display: flex !important;
+  display: flex!important;
```
Comment (vinyll) [inline static/css/around-map.css]: why removing this space?

File: static/css/around-map.css [hunk 4]
```diff
@@ -295,63 +295,34 @@ footer {
...
-  cursor: pointer;
-  width: 2rem;
-  height: 3rem;
-  border-radius: 0;
-  background-color: transparent;
-  border: transparent;
-  z-index: 2;
-  left: -0.4rem;
-  top: -2rem;
+.first-card {
```
Comment (vinyll) [inline static/css/around-map.css]: jsut curious, but why setting a class instead of using `:first-of-type` or `:first-child`

File: static/css/around-map.css [hunk 5]
```diff
@@ -295,63 +295,34 @@ footer {
...
-  z-index: 2;
-  top: -1.9rem;
+.marker {
+  fill: #297ddb;
 }
-
-.mapboxgl-popup-content p {
-  text-align: center;
+.current-marker {
+  fill: #9B9B9B;
```
Comment (vinyll) [inline static/css/around-map.css]: lowercase?

File: static/css/around-map.css [hunk 6]
```diff
@@ -295,63 +295,34 @@ footer {
...
+  /* need important to overide leaflet */
+  z-index: 9999!important;
```
Comment (vinyll) [inline static/css/around-map.css]: leaflet goes up to 1010 if I remember well. You may want to set it to that level instead.

File: static/css/main.css [hunk 7]
```diff
@@ -91,7 +91,7 @@ body > header nav a {
 .avatar {
   cursor: pointer;
 }
-.avatar img {
+.avatar-img {
```
Comment (vinyll) [inline static/css/main.css]: why setting a class for the img?

File: static/css/trips-map.css [hunk 8]
```diff
@@ -1,24 +1,20 @@
-h1 {
-  margin-bottom: 3rem;
-}
-
 #map {
   width: 100%;
-  min-height: 500px;
+  min-height: 62rem;
+  top: -3.5rem;
+  z-index: 0;
```
Comment (vinyll) [inline static/css/trips-map.css]: `0`? That's the browser's default

File: static/css/trips-map.css [hunk 9]
```diff
@@ -1,24 +1,20 @@
...
+}
+
+svg {
+  fill: #279E6A;
```
Comment (vinyll) [inline static/css/trips-map.css]: lowercase

File: static/js/around-map.js [hunk 10]
```diff
@@ -1,50 +1,70 @@
...
+// POIS creation
 const worker = new Worker('/static/js/webworker-around.js')
 let pois = []
 
 function addPoi (poi) {
-  const el = document.createElement('div')
-  el.classList.add('marker')
-  el.id = poi.id
+  const marker = L.marker(poi.location, {icon: icon, alt: poi.name}).addTo(map)
+  marker._icon.setAttribute('id', poi.id)
```
Comment (vinyll) [inline static/js/around-map.js]: Is it the only way to access the icon? Most of the time, a property that starts with a `_` should not be call as is. Its "private" hence hackish. No big deal though.

File: static/js/trips-map.js [hunk 11]
```diff
@@ -1,81 +1,89 @@
...
+  iconSize: [30, 50],
+  iconAnchor: [15, 42],
+  popupAnchor: [0, -40],
+  viewBox: '300 100 1200 1200',
+  path: 'm 768,896 q 0,106 -75,181 -75,75 -181,75 -106,0 -181,-75 -75,-75 -75,-181 0,-106 75,-181 75,-75 181,-75 106,0 181,75 75,75 75,181 z m 256,0 q 0,-109 -33, -179 L 627,-57 q -16,-33 -47.5,-52 -31.5,-19 -67.5,-19 -36,0 -67.5,19 Q 413,-90 398, -57 L 33,717 Q 0,787 0,896 q 0,212 150,362 150,150 362,150 212,0 362,-150 150,-150 150,-362 z',
+  transform: 'matrix(1,0,0,-1,364.47458,1270.2373)'
+}
 
-  lineCoords.push(point.position)
+class Icon extends L.Icon {
```
Comment (vinyll) [inline static/js/trips-map.js]: seems like copy and paste. We'll need to refactor this code into one single spot

File: static/js/trips-map.js [hunk 12]
```diff
@@ -1,81 +1,89 @@
...
+  createIcon() {
+    const marker = document.createElement('div')
+    const options = this.options
+    marker.innerHTML = `<svg xmlns:cc="http://creativecommons.org/ns" xmlns:svg="http://www.w3.org/2000/svg" version="1.1"
+      viewBox="${options.viewBox}"
+      width="${options.iconSize[0]}px"
+      height="${options.iconSize[1]}px">
+      <g transform="${options.transform}"><path d="${options.path}"/></g>
+    </svg>`
+    this._setIconStyles(marker, 'icon')
```
Comment (vinyll) [inline static/js/trips-map.js]: is there no method that doesn't start with a `_`?

File: static/js/trips-map.js [hunk 13]
```diff
@@ -1,81 +1,89 @@
...
 
-  new mapboxgl.Marker(el, {offset:[0, -30]})
-      .setLngLat(current_location.reverse())
-      .setPopup(popup)
-      .addTo(map)
-}
+lineCoords.push(lastPosition.position)
+
+// Previous trips
+icon.options.className = "previous-marker"
```
Comment (vinyll) [inline static/js/trips-map.js]: single quotes

File: templates/around/map.html [hunk 14]
```diff
@@ -3,10 +3,14 @@
 {% block title %}Around you{% endblock %}
 
 {% block extra_css %}
-  <link rel=stylesheet href=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.css >
-  <link rel=stylesheet href={{ url_for('static', filename='css/trips-map.css') }}>
   <link rel=stylesheet href={{ url_for('static', filename='css/around-map.css') }}>
-  <script src=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.js></script>
+  <!-- leaflet -->
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.2.0/dist/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin=""/>
```
Comment (vinyll) [inline templates/around/map.html]: you very obviously copy and pasted. Code I've never seen before 😮 
You may remove quotes where not required ;)

File: templates/trips/map.html [hunk 15]
```diff
@@ -9,23 +9,18 @@
 {% endblock %}
 
 {% block extra_css %}
-<link href=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.css rel=stylesheet>
 <link rel=stylesheet href={{ url_for('static', filename='css/trips-map.css') }}>
-<script src=https://api.mapbox.com/mapbox-gl-js/v0.34.0/mapbox-gl.js></script>
+<!-- leaflet -->
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.2.0/dist/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin=""/>
```
Comment (vinyll) [inline templates/trips/map.html]: same

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/js/around-map.js (hunk 10 above)
Comment (fuentesloic) [reply static/js/around-map.js]: I try many many many way without suceed.
the smart should be: `L.marker(poi.location, {icon: icon, id: poi.id})` that work for alt but not for the `id` and I dont know why :s

**Comment by fuentesloic** [comment]
@vinyll could you check this code? for me it's done (for this part)
I refacto the creation map and marker cache (I hope you was thinking to do as I did ^^)

@AymericFaivre @newick : what do you think about the map and color marker ?
Like all know it, I'm not good with this kind of things, we have some suggestion from: 
[mapbox](https://www.mapbox.com/maps/#ready-to-use)
feel you free to propose other source if you want :)
let me know (help me plzzzzz ^^) wich one is better according your designer soul :)

For the marker I have 3 colors:
- blue for a standard marker,
- orange (eurk) when you hover a card (that change the color of its pair marker)
- green when you select the pair poi marker/card

thinks guyss :)

File: static/js/around-map.js (hunk 10 above)
Comment (vinyll) [reply static/js/around-map.js]: Maybe `id` is a reserved key name

File: static/components/PoiCard.js [hunk 16]
```diff
@@ -14,7 +14,7 @@ class PoiCard extends React.Component {
     }
     // cf: https://facebook.github.io/react/docs/handling-events.html
     this.hoverCard = this.hoverCard.bind(this)
-    this.clickCard = this.clickCard.bind(this)
+    // this.clickCard = this.clickCard.bind(this)
```
Comment (vinyll) [inline static/components/PoiCard.js]: remove the line if it's unused

File: static/components/PoiCard.js [hunk 17]
```diff
@@ -57,14 +57,14 @@ class PoiCard extends React.Component {
...
-    urlFor(this.props.details.id)
-  }
+  // clickCard() {
+  //   let zoomLevel = 11
+  //   if(this.props.details.id === getHash() && map.getZoom() < 14) {
+  //     zoomLevel = 14
+  //   }
+  //   moveTo(this.props.details.location, zoomLevel)
+  //   urlFor(this.props.details.id)
+  // }
```
Comment (vinyll) [inline static/components/PoiCard.js]: remove if unused

File: static/css/around-map.css [hunk 18]
```diff
@@ -296,28 +296,30 @@ footer {
...
-  border: 0.2rem solid #3ba77c;
-}
+/* marker */
 
 .marker {
   fill: #297ddb;
 }
+
+/* current marker/card hovering */
+.current-card h2{
```
Comment (vinyll) [inline static/css/around-map.css]: space before `{`
Where is the _hovering_ that the comment says?

File: static/js/around-map.js [hunk 19]
```diff
@@ -123,24 +123,20 @@ function findPoi (id) {
...
+  return twins = [
+    { class: 'marker', div: marker },
+    { class: 'card', div: card }
+  ]
+}
+
+function highlight (poi_id) {
+  getTwins(poi_id).forEach(el => {
+    el.div.classList.toggle(`current-${el.class}`)
+  })
```
Comment (vinyll) [inline static/js/around-map.js]: I guess I'd replace the whole `getTwins` function and the forEach function with the simpler:
```js
document.getElementById(poi_id).classList.toggle('current-marker')
document.querySelector(`#card-${poi_id}`).classList.toggle('current-card')
```

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/components/PoiCard.js [hunk 20]
```diff
@@ -45,7 +45,7 @@ class PoiCard extends React.Component {
   }
 
   renderDistance(distance) {
-    if(!distance) return ''
+    if(distance < 15) return 'close to you'
```
Comment (vinyll) [inline static/components/PoiCard.js]: Why not taking a value like 100m?

File: static/css/around-map.css [hunk 21]
```diff
@@ -311,13 +310,22 @@ footer {
   color: #9b9b9b;
 }
 .current-marker {
-  /* need 201 to overide leaflet native at 200  */
+  /* need 201 with important to overide leaflet native at 200  */
   z-index: 201 !important;
```
Comment (vinyll) [inline static/css/around-map.css]: Are you sure you're not supposed to go up to 1010? (leaflet's top layer)

File: static/js/around-map.js [hunk 22]
```diff
@@ -84,9 +84,10 @@ window.onhashchange = _ => {
 }
 
 map.on('click', event => {
-  const poi = findPoi(event.originalEvent.target.id)
+  const poi = findPoi(event.originalEvent.target.ownerDocument.activeElement.id)
+  console.log(poi);
```
Comment (vinyll) [inline static/js/around-map.js]: remove

File: static/css/around-map.css [hunk 23]
```diff
@@ -280,59 +290,43 @@ footer {
...
 
 /* marker */
-
 .marker {
   fill: #297ddb;
 }
+.hash-marker {
+  fill: #3ba77c;
+  /* leaflet native max at 1010 */
+  z-index: 1010!important;
```
Comment (vinyll) [inline static/css/around-map.css]: space before `!important` (for consistency)

File: static/css/_article_list.css [hunk 24]
```diff
@@ -45,6 +45,13 @@
   display: block;
   margin-bottom: 0.2rem;
 }
+.meta a img {
+  display: inline-block;
+  vertical-align: middle;
+  border: 1px solid #666;
```
Comment (vinyll) [inline static/css/_article_list.css:51]: you 😈 `666`

File: static/js/map.js [hunk 25]
```diff
@@ -0,0 +1,45 @@
...
+  iconSize: [30, 50],
+  iconAnchor: [15, 42],
+  popupAnchor: [0, -40],
+  className: 'marker',
+  viewBox: '300 100 1200 1200',
+  path: 'm 768,896 q 0,106 -75,181 -75,75 -181,75 -106,0 -181,-75 -75,-75 -75,-181 0,-106 75,-181 75,-75 181,-75 106,0 181,75 75,75 75,181 z m 256,0 q 0,-109 -33, -179 L 627,-57 q -16,-33 -47.5,-52 -31.5,-19 -67.5,-19 -36,0 -67.5,19 Q 413,-90 398, -57 L 33,717 Q 0,787 0,896 q 0,212 150,362 150,150 362,150 212,0 362,-150 150,-150 150,-362 z',
+  transform: 'matrix(1,0,0,-1,364.47458,1270.2373)'
+}
+
+class Icon extends L.Icon {
```
Comment (vinyll) [inline static/js/map.js:30]: 👍

**Review by vinyll — APPROVED** [review]

File: templates/around/map.html [hunk 26]
```diff
@@ -5,8 +5,8 @@
 {% block extra_css %}
   <link rel=stylesheet href={{ url_for('static', filename='css/around-map.css') }}>
   <!-- leaflet -->
-  <link rel=stylesheet href=https://unpkg.com/leaflet@1.2.0/dist/leaflet.css integrity=sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==/>
-  <script src="https://unpkg.com/leaflet@1.2.0/dist/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log=="></script>
+  <link rel=stylesheet href=https://unpkg.com/leaflet@1.2.0/dist/leaflet.css integrity=sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ== crossorigin=/>
```
Comment (vinyll) [inline templates/around/map.html]: `crossorigin=`?
I'm not sure what this does, however it should have quotes `"` for it's empty value, or have no `=` at all!

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/components/Comments.js [hunk 27]
```diff
@@ -15,12 +15,11 @@ class Comments extends React.Component {
...
-      .map(key =>
-        <li><p>{this.state.comments[key]}</p></li>
-      )
-    // webnomade API have two cases for empty comments - issue #96
-    if(comments.length === 0 || comments[0] == [""]) {
+    .keys(this.state.comments)
+    .map(key =>
+      <li><p>{this.state.comments[key]}</p></li>
+    )
+    if(comments[0] == [""]) {
```
Comment (vinyll) [inline static/components/Comments.js]: single quotes 😛

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/components/Comments.js (hunk 27 above)
Comment (fuentesloic) [reply static/components/Comments.js]: crap! :)

File: templates/around/map.html (hunk 26 above)
Comment (fuentesloic) [reply templates/around/map.html]: without `=` we have an error

File: static/css/_article_list.css (hunk 24 above)
Comment (fuentesloic) [reply static/css/_article_list.css:51]: mouahahahahah

File: templates/around/map.html (hunk 26 above)
Comment (vinyll) [reply templates/around/map.html]: In which case you could then say `crossorigin=""`.
What's the error message BTW?

**Comment by newick** [comment]
@fuentesloic could we have some screenshots ? :D

**Comment by fuentesloic** [comment]
- blue for a standard marker,
- orange (eurk) when you hover a card (that change the color of its pair marker)
- green when you select the pair poi marker/card

output_jzx4gg

**Comment by newick** [comment]
First of all, add a white border on every card, like that it won't move on hover.
Remember that hover don't work on mobile ;)

I'm ok with the idea, for the color, let's see with @AymericFaivre 😛

**Comment by fuentesloic** [comment]
@newick thinks for it, concerning the mobile version there are not cards, just the map with a popup displayed when the user click on them

**Comment by AymericFaivre** [comment]
Hey !
When you hover a card, maybe you could put all the none selected/hover pin in grey and keep the over pin in blue?

So the 2 pins (selected and hover) would be well localised. 

Is it possible? Or a good idea?

**Comment by vinyll** [comment]
All good for me. I let you deal with the gfx guyz. @newick / @AymericFaivre: I'll let you approve this PR when you agree with @fuentesloic 😗

**Comment by fuentesloic** [comment]
@AymericFaivre : yes sure it's possible, should we modify  the changement on card according the actions ?

**Comment by vinyll** [comment]
Note that clicking twice on the same icon keeps it orange.
I would accuse `toggle`…

screen shot 2017-09-05 at 9 46 16 am

**Comment by AymericFaivre** [comment]
@fuentesloic yep ;)
If it's possible to have a screenshot to be sure it's a good idea it would be perfect :)

**Comment by fuentesloic** [comment]
@AymericFaivre : what do you think ?
output_gjqhvl

**Comment by fuentesloic** [comment]
@vinyll that was the toggle ^^

File: static/css/around-map.css [hunk 28]
```diff
@@ -44,6 +44,7 @@ main {
   background-color: #fefefe;
   cursor: pointer;
   margin-top: 2rem;
+  border: 0.2rem solid #fefefe;
 }
 .card:hover {
   box-shadow: 0 1rem 2rem 0 rgba(0,0,0,0.2);
```
Comment (vinyll) [inline static/css/around-map.css]: spaces after commas :p

File: static/css/around-map.css [hunk 29]
```diff
@@ -323,19 +324,25 @@ footer {
...
+  fill: #8f8f8f;
+  /* leaflet native max at 1010 */
+  z-index: 1010 !important;
 }
 .current-marker {
-  fill: #fc8428;
-  /* current marker/card hovering with 1011 */
-  z-index: 1011 !important;
+  fill: #297ddb;
+  /* hash marker hovering with 1012 */
```
Comment (vinyll) [inline static/css/around-map.css]: could you rephrase this comment?

File: templates/trips/map.html [hunk 30]
```diff
@@ -15,9 +15,8 @@
 <script src=https://unpkg.com/leaflet@1.2.0/dist/leaflet.js integrity=sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log== crossorigin=></script>
 <!-- leaflet.leaflet-locatecontrol -->
 <link rel=stylesheet href=//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css>
-<link rel=stylesheet href=https://domoritz.github.io/leaflet-locatecontrol/dist/L.Control.Locate.min.css/>
 <script src=https://domoritz.github.io/leaflet-locatecontrol/dist/L.Control.Locate.min.js charset="utf-8"></script>
```
Comment (vinyll) [inline templates/trips/map.html]: no quotes

File: templates/around/map.html [hunk 31]
```diff
@@ -3,14 +3,13 @@
...
-  <link rel=stylesheet href=//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css>
-  <link rel=stylesheet href=https://domoritz.github.io/leaflet-locatecontrol/dist/L.Control.Locate.min.css/>
-  <script src=https://domoritz.github.io/leaflet-locatecontrol/dist/L.Control.Locate.min.js charset="utf-8"></script>
+<link rel=stylesheet href={{ url_for('static', filename='css/around-map.css') }}>
+<!-- leaflet -->
+<link rel=stylesheet href=https://unpkg.com/leaflet@1.2.0/dist/leaflet.css integrity=sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ== crossorigin=""/>
+<script src=https://unpkg.com/leaflet@1.2.0/dist/leaflet.js integrity=sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log== crossorigin=""></script>
+<!-- leaflet.leaflet-locatecontrol -->
+<link rel=stylesheet href=//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css>
+<script src=https://domoritz.github.io/leaflet-locatecontrol/dist/L.Control.Locate.min.js charset="utf-8"></script>
```
Comment (vinyll) [inline templates/around/map.html]: no quotes

**Comment by vinyll** [comment]
`toggle` is here again!
clicking on a gray marker makes it green
clicking on a green marker makes it blue
clicking outside of a green marker removes the green one to select the one clicked and make it green.
You end with a bunch of grey markers, some blue markers, and one green.
Nice rainbow. What's the meaning of the blue?

screen shot 2017-09-05 at 7 13 18 pm

**Comment by AymericFaivre** [comment]
I thinks it's good to me @fuentesloic , if we can remove the bug that @vinyll saw ;)

**Comment by fuentesloic** [comment]
@vinyll It's perfectly works on my laptop, could you re build the preact app plz

**Comment by fuentesloic** [comment]
@vinyll : I just forget to change the click action ...
Edit: worse than that I myself create the bug ^^ It's solve with the next commit

File: static/css/around-map.css (hunk 28 above)
Comment (fuentesloic) [reply static/css/around-map.css]: I have to install a plugin to see this fucking invisible spaces (if you have a name ...)

Comment (vinyll) [reply static/css/around-map.css]: I just meant `box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.2);` ;)

**Comment by fuentesloic** [comment]
I think all is good now, another try required ;)

**Comment by AymericFaivre** [comment]
Thanks !

**Review by vinyll — APPROVED** [review]
Just need to squash most commits to have only a few
